flang/lib/Lower/OpenMP.cpp

```diff
@@ -202,7 +206,7 @@ genOMP(Fortran::lower::AbstractConverter &converter,
 template <typename Directive>
 static void createParallelOp(Fortran::lower::AbstractConverter &converter,
                              Fortran::lower::pft::Evaluation &eval,
-                             const Directive &directive) {
+                             const Directive &directive, bool isCombined) {
```

Comment:
I think `isCombined` could be converted to a template argument here. (Quick scan suggests it is always passed as a constant.)

Reply:
OK, that sounds like a good idea. Done.